android/src/main/java/com/magicleap/magicscript/scene/nodes/layouts/manager/PageViewLayoutManager.kt

```diff
@@ -36,12 +36,7 @@ class PageViewLayoutManager : VerticalLinearLayoutManager<PageViewLayoutParams>(
                 node.hide()
             }
         }
-        if (children.size > visiblePage) {
-            val activeChild = children[visiblePage]
-            val bounds = mapOf(0 to childrenBounds[visiblePage]!!)
-            super.layoutChildren(layoutParams, listOf(activeChild), bounds)
-        }
-
+        super.layoutChildren(layoutParams, children, childrenBounds)
```

Comment:
We cannot to that, because layout bounds will be incorrect - we must layout only one child for page view.